db/migrate/20161229204445_set_default_content_for_posts.rb

```diff
@@ -0,0 +1,8 @@
+class SetDefaultContentForPosts < ActiveRecord::Migration
+  def up
+    change_column_default :posts, :content, ''
```

Comment:
This shouldn't need a migration at all, though? Do we just need a .to_s somewhere, or something, this isn't something that ever happens in the actual correct user flow or even really in tests.